testcases: adopt newer format of cpu.pressure

kamalesh-babulal opened the pull request:
This patchset adds support to recognize the newer output of 
`cpu.pressure` on cgroup v2, which got introduced by the Kernel 
commit 0e94682b73bf ("psi: introduce psi monitor").  The newer
version of kernels with the change displays an additional line to
the existing output:
```
# cat cpu.pressure
some avg10=0.00 avg60=0.00 avg300=0.00 total=0
full avg10=0.00 avg60=0.00 avg300=0.00 total=0
```
this change in the kernel affects, multiple test cases, and each
commit fixes affected test cases individually.

kamalesh-babulal commented:
**V2 Changes**
- Fix the commit title of the third commit.

drakenclimber commented on ftests/009-cgget-g_flag_controller_only.py:
```diff
@@ -80,14 +96,18 @@ def test(config):
-                              len(out.splitlines()))
-                )
-        return result, cause
+    if (len(out.splitlines()) != len(expected_out.splitlines()) and
```
Will this catch the case where a cgroup v1 system has a different number of lines?

kamalesh-babulal replied:
@drakenclimber Thanks for the review.  Agree, it will not. Let me try and re-spin V3, addressing it.

kamalesh-babulal commented:
**V3 Changes**
- Addressed the review comments from @drakenclimber to catch erroneous output for V1 too, which was skipped in the previous version of the patch.

drakenclimber commented:
Pushed to main.  Thanks, @kamalesh-babulal
